checkstyle/checkstyle-suppressions.xml

```diff
@@ -141,6 +141,7 @@
     <suppress checks="LineLengthCheck" files="ContactCreateCNNICExtensionTest" />
     <suppress checks="LineLengthCheck" files="ContactInfoCNNICResponseExtensionTest" />
     <suppress checks="LineLengthCheck" files="ContactUpdateCNNICExtensionTest" />
+    <suppress checks="LineLengthCheck" files="DomainInfoCDNResponseExtensionTest" />

```

Comment:
This is unnecessary. Break the xml in the test into smaller lines. Makes it easier to read the xml what you expect.